---
title: Vänteläge med Cold ger "DataStoreException-posten finns inte"
description: Beskrivning
solution: Experience Manager
product: Experience Manager
applies-to: Experience Manager
keywords: KCS,AEM,coldstandby,DataStoreException
resolution: Resolution
internal-notes: https://issues.apache.org/jira/browse/OAK-8202,https://jira.corp.adobe.com/browse/GRANITE-11668
bug: false
article-created-by: Vincent Frey
article-created-date: 5/12/2022 3:39:59 PM
article-published-by: Vincent Frey
article-published-date: 5/12/2022 3:40:16 PM
version-number: 2
article-number: KA-15610
dynamics-url: https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=8e6000c7-09d2-ec11-a7b5-0022480a8683
exl-id: d37d0338-be4f-4660-a4b2-cd1179717e07
source-git-commit: e8f4ca2dd578944d4fe399074fab461de88ad247
workflow-type: tm+mt
source-wordcount: '311'
ht-degree: 0%

---

# Vänteläge med Cold ger &quot;DataStoreException-posten finns inte&quot;

## Beskrivning


<b>Problem:</b>
Kylvänteläge med dedikerad fil Datastore genererar en varning om att ett sådant blobbbud saknas som det vid 1.

<b>Information:</b>
Det här är bara en varning, inte ett fel. Det beror på att datalagret inte har något sätt att verifiera om en binär fil finns lokalt, förutom genom att försöka läsa in blob-ID.
Om ID:t läses in visas inte den här varningen i loggarna. Om det inte hittas vet dock vänteläget att det måste synkroniseras, så varningen 1 visas i loggarna.


1: *\*WARN\* standby-run-1 org.apache.jackrabbit.oak.plugins.blob.datastore.DataStoreBlobStore Ett fel inträffade när byte lästes in från team när ID f68a92b34779bbd194184251ec hämtades 916370f3ebd301d6e32a8980aad070d57a1845#25522
<br>java.util.concurrent.ExecutionException: java.io.IOException: org.apache.jackrabbit.core.data.DataStoreException: Post f68a92b34779bbd194184251ec916370f3ebd301d6e32a8980aad070d57a18 45 finns inte
<br>    at org.apache.jackrabbit.oak.cache.CacheLIRS$Segment.load(CacheLIRS.java:1017) org.apache.jackrabbit.oak-core-spi:1.10.3
<br>...
<br>Orsakad av: org.apache.jackrabbit.core.data.DataStoreException: Posten cabdc49c6562afe71da796b57c2d1a6a3495026dfafff4d1073f1935a2c49c20 finns inte
<br>    at org.apache.jackrabbit.core.data.AbstractDataStore.getRecord(AbstractDataStore.java:59)*


## Upplösning


Detta fungerar som avsett och tyder inte på något problem med File DataStore.

<b>Så här bekräftar du:</b>

- Exempel på blob-ID som saknas i loggarna: &quot;cabdc49c6562afe71da796b57c2d1a6a3495026dfafff4d1073f1935a2c49c 20 tum
- För detta blob-ID ska filsystemsadressen vara: &quot;ca/bd/c4/9c6562afe71da796b57c2d1a6a3495026dfafff4d1073f1935a2c4 9c20&quot;
- Sök i filsystemet i vänteläge och kontrollera om den namngivna filen finns i datalagret.
- Använd listkommandot (ls -ald) i /database/datastore: &quot;ls -ald ca/bd/c4/9c6562afe71da796b57c2d1a6a3495026dfafff4d1073f193 5a2c49c20&quot;
- Du bör kunna se filen och bekräfta att blob-ID:t faktiskt inte saknas
